modules/rostests/apitests/shell32/IShellFolderViewCB.cpp

```diff
@@ -246,11 +246,15 @@ class CFolderViewCB :
     public IShellFolderViewCB
 {
 public:
+    CFolderViewCB(void) :
+        m_RefCount(1)
```

Comment:
Why did you add a refcount here?

Reply:
To make it behave like a refcounter. Maybe I'm getting something wrong here, but IIUC this is not a callback, but an actual reference counter for the callback object. I don't really understand the referencing behavior (like why it's not dereferenced, when a new callback object is set), but this seems to make much more sense.

Reply:
The object is a callback, which has a reference counter,
but since I didnt want to have it dynamically destruct I left it at a fixed reference count.

Reply:
But the static counters were not reference counters, but call counters.